src/table/settings.ini

```diff
@@ -2830,6 +2831,19 @@ strval   = STR_CONFIG_SETTING_ZOOM_LVL_OUT_2X
 proc     = ZoomMinMaxChanged
 startup  = true
 
+[SDTC_VAR]
+var      = gui.sprite_zoom_min
+type     = SLE_UINT8
+flags    = SLF_NOT_IN_SAVE | SLF_NO_NETWORK_SYNC
+guiflags = SGF_MULTISTRING
+def      = ZOOM_LVL_MIN
+min      = ZOOM_LVL_MIN
+max      = ZOOM_LVL_OUT_4X
```

Comment:
Does this setting really work at "OUT_4x"? I would have expected that _NORMAL sprites are required.

Reply:
I might have misinterpreted something, but the zoom levels are a bit oddly named (and ordered) - here is line 247 from the v2 sprite loader:

`static const ZoomLevel zoom_lvl_map[6] = {ZOOM_LVL_OUT_4X, ZOOM_LVL_NORMAL, ZOOM_LVL_OUT_2X, ZOOM_LVL_OUT_8X, ZOOM_LVL_OUT_16X, ZOOM_LVL_OUT_32X};`

`ZOOM_LVL_OUT_4X` is equal to `2U`, then `NORMAL` is `0U`, `2X` is `1U` and then `8X` (the first "zoom out" level) is `3U` and so on.

As far as I can tell (and I could be wrong), ZOOM_LVL_OUT_4X is what we think of as normal "1x" sprites, ZOOM_LVL_NORMAL is "4x zoom in", ZOOM_LVL_OUT_2X is "2x zoom in" and then the others are the  regular zoom levels. At least, that's consistent with what I found when playing around with this, although it could be I've misinterpreted what's going on in some way.

At any rate, in game you get a choice between "1x", "2x" and "4x" which enable/disable the alternative sprites as you'd expect.

Reply:
Indeed, oddly named. Thanks :)